hedera-node/src/main/java/com/hedera/services/state/merkle/internals/BitPackUtils.java

```diff
@@ -90,6 +119,18 @@ public static int codeFromNum(long num) {
 		return (int) num;
```

Comment:
If the `num` value IS `0xFFFFFFFF` here, will the return value be overflowed?

Reply:
The "codes" are (signed) `int`s in the range `[Integer.MIN_VALUE, Integer.MAX_VALUE]`

Reply:
I see. In reality for this case, there will never be a chance for the overflow to happen.